06.bowling_object/game.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require './frame'
+
+class Game
+  def initialize(pinfall_text)
+    @frames = generate_pinfall_frames(pinfall_text)
+  end
+
+  def main
+    total_score = @frames.each_with_index.sum { |frame, idx| frame.scores + calculate_bonus(idx, frame) }
```

Comment:
```suggestion
    total_score = @frames.sum { |frame| frame.scores(frames) }
```

点数計算のロジックをGameクラスではなくFrameクラスに寄せてみましょう。

なお、フレーム番号（idx）はFrameをnewするときに渡すイメージです。